.github/how_to_compile_gudhi_in_a_conda_env.md

```diff
@@ -0,0 +1,92 @@
+# Install a conda development environment to compile GUDHI
+
+## Install miniconda
+
+Download the [installer](https://docs.conda.io/en/latest/miniconda.html) required by your system and follow the [instructions](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+## Create a dedicated environment
+
+```bash
+conda install -c conda-forge mamba  # installation with mamba is faster
+conda create --name gudhi
+conda activate gudhi
+mamba install -c conda-forge python cmake doxygen eigen cgal-cpp
+```
+
+Some of the requirements are in the gudhi repository (please refer to
+[how to use github to contribute to gudhi](how_to_use_github_to_contribute_to_gudhi.md)).
+In the gudhi repository - let's call it `/workdir/gudhi` i.e. - once submodules are initialised:
```

Comment:
Maybe mention the need to clone the whole gudhi repo (I guess it's not always straightforward) - including submodules - and once they are initialized...

Reply:
Ok, you did the same remark than @mglisse , so I precised it on [5839625](https://github.com/GUDHI/gudhi-devel/pull/607/commits/5839625a253fbb54b40205da1a3e2811b29a1be2)